### DISCUSSION TITLE
Who's hiring? (Jan 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#32937] [Ideas]
# What is this?
This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

*If you're looking for last month's discussion, see #32093. At least one company has [hired someone](https://twitter.com/Nutlope/status/1475929984174895105) as a result of it!* 

If you're a developer looking to post about themselves, use #32938.

**Comment by duosana** [comment]
Sana Benefits | Senior Full-Stack Engineer | Remote (US Timezone) | Full-time | $120K - 170K+ base + generous equity | [Link](https://jobs.lever.co/sanabenefits/943da474-5feb-4e67-bb3d-2a9be0f6b127?lever-via=1BLCQyk8VY)

We are looking to onboard several new mid/senior level full-stack engineering. We are trying to improve the health insurance industry by helping small businesses and medium sized companies. You can read more about us here: https://www.builtinaustin.com/2020/09/10/sana-benefits-raises-20m-series-a

We have a wonderful community of developers and a culture of growth and transparency. We enjoy supporting/mentoring each other and promoting from within. We offer competitive salaries and amazing benefits - we are a benefits company after all!

We are constantly adding new positions! You can check our all of our job listings [here](https://jobs.lever.co/sanabenefits?lever-via=1BLCQyk8VY).

For any questions, please reach out to me (#Dewo6376) or just comment your information below!

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by elsigh** [comment]
Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)

Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.

Feel free to apply using the link above.

**Reply by EnshaednHiker** [reply]
Would a frontend engineer be a good fit for any roles or do you need to have that backend experience?

**Comment by mmmeff** [comment]
🍿 IMDb.com | Full-Stack/Front-End Engineer | Seattle or Remote (US) | Full-time |  $100-$150k base (level-based) + generous $AMZN equity | [Link](https://www.amazon.jobs/en/jobs/1261725/front-end-engineer-imdb?no_int_redir=1)

> IMDb is embarking on a bold re-imagining of the consumer experience built on GraphQL, NextJS and React. We are building an entirely new distributed architecture to support iOS, Android and Web clients. We are seeking an experienced frontend engineer for the web engineering team who will work with technology, product and design leaders to deliver performant and delightful experiences that help entertainment fans explore the world of movies and shows and decide what to watch next.

> IMDb is a lot like working for a smaller company with all of the big company benefits. We know each other, we like each other, we share a passion for movies and TV. We believe in hands on career development, setting clear goals and expectations, and being transparent with opportunities to grow your career.

Hiring front-end-oriented folks to help my team migrate IMDb.com to Next.js. We're rebuilding it all, one page at a time, on Next and GraphQL, redesigning and reimagining along the way. Interested and want to chat? Drop me an email at matfrey@imdb.com and we can set up a referral.

**Reply by shambashib20** [reply]
I am well versed with ReactJS but a beginner in NextJs. Can I apply for the same now?

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by jaxomlotus** [comment]
BurnerPage | Full-Stack Engineer | NYC or Remote (US only) | Full-time | $115-175k base / benefits / early equity | [Link](https://docs.google.com/document/d/1264EHpy8AKf_tar7wsFDq_hlHP9WrdTdF7A9qUbPAVg/edit?usp=sharing)

BurnerPage is a stealth startup founded by a repeat entrepreneur who sold his last company to Adobe. We are VC backed and looking to make hires 2-5 as we bring our product to market. 

The above link is for [front-end devs](https://docs.google.com/document/d/1264EHpy8AKf_tar7wsFDq_hlHP9WrdTdF7A9qUbPAVg/edit?usp=sharing), but we're looking for [backend](https://docs.google.com/document/d/1y09qDXdosWNrI8cYoOYE9V5FP6s5g3LHC-qkE99jMZs/edit) / fullstack and [machine learning engineers](https://docs.google.com/document/d/1QYJKwhTrlb5hMb_crKffPpQtiyr_plAr7aVvYhAMhqg/edit) as well (with different base pay for those positions depending on experience). BurnerPage is building the future of webpages, harnessing the power of AI and Machine Learning to automatically build webpages and tailor the viewer experience to the individual. [Note: The [current product](https://burnerpage.com/) does not yet advertise this purpose - it was just a manual prototype we launched that hit [#1 on ProductHunt](https://www.producthunt.com/posts/burner-page), but the stealth AI based product is the real focus that we're hiring for].

I'm the founder and looking to find amazing folks who love working in Next.js. Feel free to reach out to me directly to apply at avi -(a)- burnerpage.com

**Reply by prmlee** [reply]
Hi, jaxomlotus! Nice to meet you. I am a full stack developer with 6 years experience in node.js.  I am so interested in your backend/full stack role. I hope we will meet soon. Thank you! discord id: Po#9031

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by sb2nov** [comment]
corise |  Full-Stack Engineer | Remote (US) | Full-time | $100-150k base | [Link](https://corise.notion.site/Full-Stack-Software-Engineer-9e7e92a5954b47eea42655ea3a220d6b)

We are a small but mighty team working on a mission to upskill the world's workforce. 

Over the next decade, 1 billion jobs will be reimagined by the development of new technologies. Companies need to reskill their employees, but they lack the know-how to do so. Individuals need to reskill throughout their careers to keep pace with changing requirements, but their learning options are insufficient.

We are building a better solution using social learning and technology to drive outcomes at scale. Our project-based learning platform empowers employees to make step function transformations in their careers and companies to upskill their workforces.

Feel free to apply using the link above.

**Reply by jamesyfyp** [reply]
Hello, my name is James I'd be super interested in working for your company. I am a self-taught developer whose been working with react for the last few years. Let me know if your organization is looking for juniors.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by diegoserranor** [comment]
Wayru | Backend Engineer (smart contracts big+) | Remote (LATAM) | [Link](https://www.linkedin.com/jobs/view/2860022154/)

We are working on a new kind of ISP for LATAM, and on a mission to close the digital divide by providing reliable and affordable Internet access.

We are building fullstack projects with Next, and are big JS/TS fans. On the smart contracts end, we are building these on Algorand, and previous experience with this blockchain is highly desirable 🥷

Apply with the link in the tag above.

**Reply by prmlee** [reply]
Hi, I am a full stack developer in web dev and blockchain. I developed the NFT marketplace sites, NFT token swapping and NFT mint sites. I can help you perfectly. You can check yesports.gg,  https://nftsrealm-bsc.netlify.app, prysm.xyz and etc. These are my past works. When developed NFT marketplaces, I was responsible for developing marketplace contract and nft contract. I hope I will get a positive answer/response soon. thank you! (discordid: Po#9031, promotionlee.bs@gmail.com)

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by bechtleag** [comment]
Bechtle AG | frontend developer | germany / remote possible | full-time | [Link](https://jobs.bechtle.com/job/Neckarsulm-Fullstack-JavaScript-Developer-%28wmd%29/732977702/)

The Bechtle web platform offers a rich B2B portfolio of IT products and services. We are a young team of out-of-the-box thinkers with a great tech stack and now looking for you/coders to help us migrate our platform to NextJs.

Besides developing with NextJs, you can contribute to a host of exciting things such as UI/UX design, headless CMS, or evaluating our Jamstack architecture. Join us as we build the future of Bechtle.

If you have any questions, please feel free to get in touch at maik.lieweries@bechtle.com, or apply directly via the link above.

We’re hiring for different engineering teams, levels and roles! Junior entries are very welcome. Please be advised that the primary language spoken across all teams is German.

We look forward to hearing from you!

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by omure** [comment]
Omure | Senior React.js Engineer | Remote (Europe, Americas) | Full-time | [Link](http://omure.com/jobs/senior-react) 

We're looking for experienced React/Next devs to join our Front-end team.

There's no legacy or client work involved and hours are flexible.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by mephysto** [comment]
**Marketforce | Senior Front End | Local | Perth, Australia | Full-time | [Link](https://marketforce.com.au/)**

Marketforce is a creative advertising agency, part of the global BBDO network. We're hiring people to strengthen the digital team. We're looking to expand our group of developers to meet the demand of our headless CMS, Next sites, and various creative digital campaign offerings. 

We're versed in: Next.JS, Typescript, GraphQL, Wordpress (as a Headless CMS), Node, AWS services.

We are a creatively lead technology team that helps transformational change, consumer engagement and better user experiences. We are a collective of specialist creative and technology experts that leverage our extensive background to help make better design decisions that enable platform capability.

Come join us in sunny West Australia, where the pandemic has hit the least. And we can still go out and hug eachother in the pub. 

Feel free to reach out to me directly, and send me a resume and some of your most recent work.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by sullman** [comment]
[Plex](https://www.plex.tv/) | Frontend Engineer | Remote (esp. UTC-8 - UTC+2) | Full-time | $90-160k base | [Link](https://grnh.se/25643b72teu)

Plex is hiring for a variety of positions, including a frontend engineer to work on web properties like [watch.plex.tv](https://watch.plex.tv/) and [www.plex.tv](https://www.plex.tv/). We recently published [a detailed blog](https://medium.com/plexlabs/plexs-2021-journey-with-next-js-and-vercel-c602851478c5) about our transition to Next.JS and Vercel if you want to get a better sense of some recent/current projects. We're a passionate group of movie and TV fans who love getting to build services and experiences to improve the modern streaming world, whether it's offering our own streaming catalog, managing your own personal media library, or discovering new content.

I've been at Plex for a decade now (:scream:) and love the people I work with as well as the ways I've been able to work with different technologies and grow through different roles.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by Sobefy** [comment]
[SOBEFY](https://sobefy.com) | Frontend Engineer | Remote (Europe, Americas) | Full-time | [LINK](https://sobefy.com)

Sobefy is hiring for a variety of positions, including a frontend engineer to work on custom web applications such as business directory locators, API integrations between Shopify and third party platforms. 

We recently built a [doctor locator](https://locator.dryeyerescue.com/) for one of our clients in Next.js using DynamoDB and AWS. The ideal candidate would help maintain and update the app with new features such as adding input fields (images, text, textarea, documents) as well as help display the data on customer facing pages. 

The ideal candidate will work with us for years to come, we are flexible when it comes to schedule and workflow and we are happy to train :)

**Reply by mzeinh** [reply]
Hello! 

Is there a job application to fill out on the link you have posted or do we email it in?

Thanks

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by yulijaflynn** [comment]
**[Witt-Gruppe](https://karriere.witt-gruppe.eu/berufsfelder/tech-development/) | (Senior) Front-End Developer | Remote (Germany only) | Full-time | [Link](https://karriere.witt-gruppe.eu/job/3412870-frontend-developer-m-w-d-remote-arbeit-moeglich/)**

Witt-Gruppe (a member of the Otto-Group) is hiring for a variety of positions, including a front-end developer to work on our Witt Commerce Cloud (WCC), which is an in-house developed platform for our web shops, that has been tailored to our needs.
Our platform is developed with state of the art technology, such as `React, Next.js, GraphQL and TypeScript`.

If you're interested or if you have any questions, please feel free to get in touch with me or apply directly via the link above.
I'm happy to see you soon in our team 🙂 

[Twitter](https://twitter.com/yuli_yulija) | [LinkedIn](https://www.linkedin.com/in/julia-w-ab90a41a1/) | [Xing](https://www.xing.com/profile/Julia_Woehrl) | [Mail](mailto:julia.woehrl@witt-gruppe.eu)

**Comment by grumart** [comment]
Village | Senior Frontend Engineer | Remote (US) or on-site in Eau Claire, WI | Full-time | $100-140k base | [Careers](https://village.recruitee.com/o/senior-frontend-engineer)

Seeking a Senior Front-end engineer with 3+ years of professional front-end experience. In this critical role as one of the first few tech hires, you'll be responsible for leading the implementation of our marketplace web-app’s front-end. You'll report directly to the CTO and will be working with stakeholders across the organization. Opportunity for significant equity compensation in a social venture startup with high-growth potential and a mission with a lasting, immediate impact.

If you're interested or have any questions, feel free to get in touch with me or apply directly via the link above.

[LinkedIn](linkedin.com/in/marta-grudzien-261b171b1)

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by gb221** [comment]
[Attio](https://attio.com) | Web Engineer | London or EU Remote | Full-time | £50-70k base + generous equity | [Apply now](https://attio.com/job/web-engineer)

- Design, implement, and improve our top-of-funnel web surfaces.
- Likely projects will include generative landing pages, template and integration libraries, and expanding Attio Atlas.
- Focus will be on highly polished and performant work that’s tightly integrated with our analytics stack and app.

Team of 13, [recently raised $7.7m](https://techcrunch.com/2021/11/24/attio-wants-to-build-the-crm-for-the-airtable-generation/?guccounter=1), will triple in size this year. Dedicated to staying an engineering-first organisation.

**Reply by jamesyfyp** [reply]
Hello, My name is James and I'm a self-taught react developer. Let me know if your organization is interested in juniors.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by braveheart-star** [comment]


**Reply by Nutlope** [reply]
Hi, please re-do your post using the format above that everyone else is using. Thanks!

**Reply by GreatWorldDev** [reply]
Hello Jack.
I can help you with Next.js, MUI, Typescript.
Could you kindly ping me Software-mars#2269?
Let's discuss more deeply.

**Comment by tvler** [comment]
**[Mirror.xyz](https://mirror.xyz/) | Frontend Engineer | Remote (global) | Full-time | [Link](https://mirrorxyz.notion.site/Frontend-Engineer-566c488643064a088c2cabb3553616b6)**

Mirror is hiring a frontend engineer to help build and maintain our user-facing suite of tools. This includes decentralized publishing tools, Web3 economic tools (crowdfunds, NFTs, splits), community management tools, governance and membership tools to empower the next wave of crypto-native creators and communities. You will also contribute to R&D for completely new product lines and experiments.

Frontend stack:

- Next.js
- Apollo/graphql
- [Degen](https://degen-xyz.vercel.app/) (our component library)

Feel free to reach out – [@ty___ler](https://twitter.com/ty___ler) or by email tyler@mirror.xyz, happy to talk more about what it's like working in web3, company culture, etc :) Very small team right now and lots of room to grow

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by karlmacklin** [comment]
[Delorean](https://www.delorean.se/) | Full-Stack Engineer | Stockholm Sweden (remote possible) | Full-time | [Link](https://jobb.delorean.se/jobs/167816-delorean-soker-fullstack-webb-utvecklare)

Delorean is a digital agency in Stockholm Sweden.
Our clients are a mix of private actors and clients from governmental procurement processes, for example we have built the offical sweden.se website for Svenska Institutet.

We're currently looking for several developers such as .NET (Umbraco), Wordpress, React, Node fullstack (Strapi etc).
Many of our projects use Next.js and it is usually our go-to tool for any project of significant size.

I'm one of the company's CTO:s and have been here since 2009 (time flies!) so I'd be happy to answer any questions you have about us/possible work here.
Either contact me directly or use the link to apply. We are currently putting up more ads to cover several roles, so even if you don't see yourself 100% described in the ad (no .NET experience - no problem), consider applying anyway!

If you don't speak Swedish, contact me to know more.
We are primarily looking for people located in Sweden, however we are still open to applications outside Sweden as well. We currently have employees in the UK for example. 

I'm reachable at kalle@delorean.se

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by Danm72** [comment]
Mawla | React Engineer | EU (remote possible) | Contract

Mawla are looking to grow our pool of frontend freelancers. We are always busy and having a network of strong freelancers means we know who to turn to when the next project comes up.
We have a boilerplate engine we use for all our next generation websites built on Tailwind, Typescript, Nextjs, Sanity.
A lot of the sanity and next work is done already by using the boilerplates so we're mostly looking for people with strong react, tailwind, typescript exp.

Rates, communication quality, timezones and availability are important to us so please DM me following this format:
-  Rates in $
-  English fluency - Yes/no
-  Timezone - we can't work with India and similar timezones right now.
-  Availability - full, part time, X hrs per week

We've three potential projects right now:

-  One react web app wizard for creating quotes
-  One SaaS company marketing website
-  One agency marketing website

We've met some of our favourite people in this slack so I'm enthusiastic we'll meet more great people.
PS our site is under construction newer one here - https://mawla-marketing.vercel.app/mawla-engine

projects@mawla.ie

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by the-kollider** [comment]
Kollider.xyz | Senior Frontend Engineer / UI/UX Designer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-125k base + equity | [Link](https://pro.kollider.xyz)

Hello everyone :wave: 

Kollider is looking for a senior frontend engineer to take the role as a lead frontend developer. 

Since UX and UI are at the heart of Kollider's value proposition a natural affinity to design is paramount. Our stack is Next.js + Typescript + Tailwind in the front and Rust in the back. You can check out the current state of the product [here](https://pro.kollider.xyz).  

If you're interested feel free [apply](https://kollider.homerun.co/senior-frontend-engineer/en) or reach out directly to me at konstantin@kollider.xyz :v:

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by VinnyFNC** [comment]
Fnatic | Full-Stack Engineer | London & Remote (UK, GER) | Full-time | [Link](https://fnatic.com/careers/a015483c-7ab6-469a-91b1-a4e64af7dae9)

We are hiring a Fullstack Engineer @Fnatic to join our Tech Team **building up the future of Esports & Entertainment !**

Your days will be filled d**eveloping engaging experiences and products across our fast-moving business.** You’ll have an eye for the fine details and the solutions you develop are the most elegant.
You'll get to **work on projects from memberships to community features, personalisation across apps** and much more!
You get excited by new techniques, problem solving regardless of the domain, and love the idea of **developing highly functional, data-rich complex-systems**.
Most of all, you're excited to join a small team where your impact is highly measurable and helps amplify a **brand loved globally by millions of fans!**

**Our Stack:**

- React, Next.js, Tailwind CSS
- Sanity CMS, Shopify, FusionAuth
- Vercel, Cloudflare & AWS
- Python, Node

I've been at the company for less than 1 years and still find plenty of opportunity for impact, and love the teammates I'm working with. 
Far, Owen, Apos, Darwin and Peter are amazing guys to hang out with and the team has delivered amazing cross platform app  within a year. 
Feel free to apply using the link above or reach out to me directly.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job post for February is up (https://github.com/vercel/next.js/discussions/33869) and you're more than welcome to repost this message there :)

**Comment by spencermarx** [comment]
### [LaFabrica](https://www.lafabrica.work/) | Full-Stack Engineers | Remote (Worldwide) | Full-time & Part-Time | [Link](https://www.lafabrica.work/developers/register)

We are looking for Fullstack Engineers to join our platform, [LaFabrica](https://www.lafabrica.work/), a crypto-driven cooperative for Software Engineers.

**The platform...**
1. Connects developers across the world directly to remote software projects paid in US Dollars 💵
2. Offers members achievement-based dividends, distributed using our very own cryptocurrency 🏦
3. Provides startups and dev agencies dedicated, thoroughly vetted, ready-to-work remote engineering teams and individual developer resources within a range of budgets 👩‍💻👨‍💻

Our Beta pool is filling up quickly, so if you’re interested, sign up soon!

Registration: ➡️ [here](https://www.lafabrica.work/developers/register) ⬅️

**Our Stack:**
Completely flexible -- from full stack Javascript to GoLang, we encourage devs from any stack to apply.

The current platform is built out using:
* Typescript
* NextJS | React | Material UI
* NestJS | Firebase
* AWS Lambda | Google Cloud Functions

Feel free to apply using the link above or reach out to me directly.

**Comment by Charlette123** [comment]
Genie AI | React Developer | London OR fully remote (UK) | Full-time | [https://apply.workable.com/genie-ai/j/8166DCED3C/](url)

Genie AI is a deep learning based software company with a mission to open source the law.

We are disrupting the legal industry by changing the business model ground up.

What Github did with open source code, Instagram and TikTok with entertainment, Airbnb with hospitality, and Uber with travel, Genie is doing with Legal contracts.

We are building a community-based AI law platform.



Requirements
About the role
We're looking for an experienced Senior React developer who has a passion for building an elegant & modern interface to solve complex real-world problems.

You are a product-focused, design thinking and curious developer who wants to turn user needs into simple yet clever solutions
You are excited to develop a community-based platform web application in React
You are interested in creating a rich, beautiful and elegant user interface to perfection
You enjoy and find it fun to build re-usable components
You are keen to protect your code with solid tests ranging from unit to automation
You care to collaborate and participate in solution design and performing experiments with the team
You are not afraid to take risks and learn advanced topics
You are eager for friendly, coherent spoken and written communication to exchange complex ideas
You build coding standards and peer review code to ensure code is neat and clean
What we need
Experience in building SaaS web-based products using;

Solid knowledge of React or NextJS framework
Typescript and Javascript
Proficient in HTML, CSS
What we would like
GraphQL
Knowledge of Design Patterns in Web development
Experience with test frameworks for unit, integration, and automation testing

What would be nice

Working with real-time collaboration applications & browser based editors
Experience with server-side rendering, web workers & service workers
Built apps with Micro-services architecture
Solid experience in responsive UI development

About our stack
Micro-services architecture, React, Next.js, Angular 12, Playwright, NodeJS, RabbitMQ, GraphQL, Redis, MongoDB, Python, AWS, Docker, Kubernetes, TensorFlow, PyTorch, ProseMirror. You're not expected to know all of this, but you will get the opportunity to learn based on your interest!


Benefits

What do Genies get?
Substantial equipment budget for you to set up a home working environment, including a top of the range laptop, monitor, wireless keyboard, mouse, and office chair
Generous stock options - we want everyone to share in our success!
Unlimited budget for business, law, or technology books
Unlimited holiday
Fully flexible working
Cycle to work scheme
Regular meditation and wellbeing sessions
Employee benefits platform where you can buy plants, food deliveries, yoga, wellbeing sessions, and more on us!
Optional pre-tax charity giving
Regular team lunches and dinners after lockdown (and plenty of social activity in the meantime!)
The opportunity to join a bright, fast-growing startup at a grassroots level and genuinely make a difference
